xml/ses_glossary.xml

```diff
@@ -80,10 +204,19 @@
     </para>
    </glossdef>
   </glossentry>
-  <glossentry><glossterm>Node</glossterm>
+  <glossentry><glossterm>&ceph; Object Storage</glossterm>
    <glossdef>
     <para>
-     Any single machine or server in a Ceph cluster.
+     The object storage “product”, service or capabilities, which consists of a
```

Comment:
```suggestion
     The object storage "product", service or capabilities, which consists of a
```

Reply:
Be careful of automatically-inserted smart/curly quotes!

Reply:
True!